### PR TITLE
Refactor, serialize jsonld on build.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Addresses issue #3.

* Switches source of truth from `contexts/context-v1.jsonld` to `js/context.js` (plain JS export).
* Serializes/publishes `contexts/context-v1.jsonld` on the build step.
* Removes `fs` usage in non-browser environments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,13 +57,21 @@ applications.
 ## API
 
 The library exports the following properties:
+- `CONTEXT_URL`
+- `CONTEXT`
```
Comment (dlongley) [inline README.md]: Do we need to say anything about packages that bundle more than one context?

**Review by dlongley — APPROVED** [review]
Seems ok, see my comment.

File: README.md (hunk 1 above)
Comment (dmitrizagidulin) [reply README.md]: @dlongley as in like "don't do it"?

Comment (dlongley) [reply README.md]: @dmitrizagidulin, yes, that's fine advice. I thought we were doing it somewhere but was mistaken.

Comment (dmitrizagidulin) [reply README.md]: (Added a comment.)

**Pull request merged**